lib/src/components/NxVulnerabilityDetails/details/MarkdownDetail.tsx

```diff
@@ -22,7 +24,9 @@ const MarkdownDetail: FunctionComponent<Props> =
     return markdown || componentSpecificMarkdown ? (
       <RenderDetail title={title}>
         {warning &&
-          <p className="nx-vulnerability-details__malicious-code-warning"><strong>{warning}</strong></p>
```

Comment:
I'd argue that regardless of styling, `<strong>` is the right semantics for this content.

Reply:
> I'd argue that regardless of styling, `<strong>` is the right semantics for this content.

Restored here: https://github.com/sonatype/sonatype-react-shared-components/pull/129/commits/700622f5b6c6cdc35917f03dc33e6ba1e3ee66fd